internal/provider/data_source_packer_image_iteration_test.go

```diff
@@ -251,6 +251,35 @@ func getIterationIDFromFingerPrint(t *testing.T, bucketSlug string, fingerprint
 	return ok.Payload.Iteration.ID, nil
 }
 
+func getBuildIDFromIteration(t *testing.T, bucketSlug, iterID, cloudProvider string) (string, error) {
```

Comment:
Where is this function called from?

Reply:
It's called in the channelAccept test: https://github.com/hashicorp/terraform-provider-hcp/pull/339/files#diff-812fcf1f8adbdbf65c25344beec1814ebf9448de4982f3e52479c3700880fdd5R202